kipoiseq/extractors/gtf.py

```diff
@@ -324,7 +326,6 @@ def _read_utr(
             on_error_warn=on_error_warn
         )
 
-        utr_df = utr_df.set_index("transcript_id")
```

Comment:
I think you need to keep this line.
- It makes sure that the groupby works by transcript id
- It gets stored in the metadata afterwards. 
That is probably also the reason why tests are failing.